.idea/.generators

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Congrats on using RubyMine!
Please add the `.idea` directory to .gitignore . It's RubyMine stuff that don't need to be committed.